### DISCUSSION TITLE
Supabase + Ory examples

### DISCUSSION THREAD

**Discussion by vinckr** [supabase/supabase#7758] [Feature Requests]
hello,
we have built some examples using Supabase as main driver and Ory for authz&authn.

- [Example using Ory Kratos, Ory Oathkeeper and Supabase](https://github.com/ory/examples/tree/master/kratos-keto-oathkeeper-supabase)
- [Example using Ory Cloud and Supabase](https://github.com/ory/examples/tree/master/supabase-ory-cloud)

do you think this would be a good addition to the official supabase examples? 
Is [this](https://github.com/supabase/supabase/edit/master/web/docs/guides/examples.mdx) the right place to add it? Can I just go ahead and open a PR?

any feedback on the examples welcome also of course 🤗

**Comment by steve-chavez** [comment]
Hey @vinckr,

> do you think this would be a good addition to the official supabase examples?

Sure! This has been requested recently actually: https://github.com/supabase/gotrue-js/issues/316

>Is [this](https://github.com/supabase/supabase/edit/master/web/docs/guides/examples.mdx) the right place to add it? Can I just go ahead and open a PR?

Please go ahead :)

**Reply by koakh** [reply]
Hello @vinckr

I update this issue https://github.com/supabase/gotrue-js/issues/316

Thanks
